src/components/shared/FormDialog.tsx

```diff
@@ -61,6 +61,7 @@ export function FormDialog({ handleClose, isOpen, title }: AddUserProps) {
 
   // validations needed
   // valid email provided
+  // why are the submit/cancel buttons listitems?
```

Comment:
🤷 Maybe whoever wrote this originally was trying to make them go on separate lines.